<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeDialog</class>
 <widget class="QDialog" name="ShapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shape</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vl1">
     <item>
      <layout class="QHBoxLayout" name="hl1" stretch="0,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="lResolution">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxResolution">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.01</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollAreaShapeWidget">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaShapeWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ShapeWidget" name="shapeWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lShapeWidgetStatus">
       <property name="text">
        <string>Shape widget status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item>
        <widget class="QLabel" name="lAmount">
         <property name="text">
          <string>Amount:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAmount"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl3" stretch="0,0,0,1,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNormalize">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lStatus">
         <property name="text">
          <string>Status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShapeWidget</class>
   <extends>QWidget</extends>
   <header>shapewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ShapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxResolution</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ShapeDialog</receiver>
   <slot>resolutionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ShapeDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAmount</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ShapeDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resolutionChanged(int)</slot>
  <slot>updateControls()</slot>
 </slots>
</ui>
